### ISSUE TITLE
Screen reader is not conveying any information of 'tree view button' of column headers.

### ISSUE THREAD

**Issue by jeneemicrosoft** [ag-grid/ag-grid#5041]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```

Current behaviour.

![image](https://user-images.githubusercontent.com/92124071/161166033-8ebbe42a-412e-4e6c-9650-1d1a8626b0fe.png)
https://www.ag-grid.com/react-data-grid/master-detail/#example-simple
1. On Master Detail grid, cannot keyboard focus and select the tree view button. 
2. Using Narrator (screen reader on Microsoft 11) also cannot convey information about the tree view button using tab or arrow keys to navigate. 

Expected behaviour. 

1. Expected to be able to get information about the tree view button using Narrator, as sight impaired users will need to be able to select the button to access the tree view menu. 

**Please tell us about your environment:**
'Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.74 Mobile Safari/537.36 Edg/99.0.1150.55'

* **AG Grid version:** X.X.X
Tested here -- https://www.ag-grid.com/react-data-grid/master-detail/#example-simple
* **Browser:**

* **Language:** 
React.

**Comment by makinggoodsoftware** [comment]
Hi,

Your ticket has been flagged as in_zendesk, it means that we recognize this as an issue worth investigating so we have moved it to our official support channel for customers (zendesk)

We will carry on there with the investigation and we will update you as soon as possible.

Thanks

**Comment by icedjello** [comment]
Hello!

Currently there's no way to implement this behavior with AG Grid. However, we agree this would be a valuable addition to the product and we have added a feature request for this.

We have added this requirement to our backlog and we are tracking it with the following reference and description:
AG-4273 - [Accessibility] Allow announcing the expanded/collapsed state of group and master rows (add the ARIA-expanded attribute)

We have no timeline for this at the moment, but this can still be picked up by our team when they're making modifications to this functional area of AG Grid. Please use the functionality currently provided by AG Grid until this is delivered at a later point.

See whether this item is selected for implementation on the NEXT RELEASE tab of the product pipeline:
https://www.ag-grid.com/ag-grid-pipeline/

Thanks again for bringing this up with us.

Regards,
